Tribler/Policies/BoostingManager.py

```diff
@@ -0,0 +1,1114 @@
+# -*- coding: utf-8 -*-
+# Written by Egbert Bouman, Mihai Capotă, Elric Milon, and Ardhi Putra Pratama H
+# pylint: disable=too-few-public-methods, too-many-instance-attributes
```

Comment:
That's what I thought as well. But it's already there before I'm writing the code. But, yeah, will do.